src/components/Navbar/index.jsx

```diff
@@ -90,11 +90,79 @@ function Navbar({ onClick, user, setUser }) {
     location.pathname !== "/dummygram/login" &&
     location.pathname !== "/dummygram/signup" && (
       <div className="app__header">
-        <Logo />
+        <img src={appLogo} alt="dummygram" className="nav_img_logo" />
```

Comment:
why are there two logos?

Reply:
Oh I get it, you are handling in css to show them conditionally. 
That's fine, also we can use mediaquery right here to show them conditionally as well and that'd make it more readable because it's confusing at first sight to see two logos.

Reply:
I think you want to say ternary operator (or media query?)

Reply:
> I think you want to say ternary operator (or media query?)

yes, ternary is obvious , I wanted to highlight useMediaQuery hook which'd be good to use there :)